packages/ai-jsx/src/lib/asr/asr.tsx

```diff
@@ -313,40 +357,73 @@ export abstract class SpeechRecognitionBase extends EventTarget {
       }
       this.streamSentMillis += (chunk.byteLength / (2 * this.manager.sampleRate)) * 1000;
     });
+    this.manager.addEventListener('vad', (evt: CustomEventInit<VoiceActivity>) => {
+      const update = evt.detail!;
+      if (!update.active) {
+        if (this.streamLastVoiceEndMillis == 0 && this.streamFirstTranscriptMillis != 0) {
```

Comment:
It probably won't matter in practice, but use `===` unless you also want to match `'0'`.

```
> '0' == 0
true
```